src/python_testing/matter_testing_support.py

```diff
@@ -265,6 +265,16 @@ def wait_for_event_report(self, expected_event: ClusterObjects.ClusterEvent, tim
         asserts.assert_equal(res.Header.EventId, expected_event.event_id, "Expected event ID not found in event report")
         return res.Data
 
+    def wait_for_event_expect_no_report(self, timeout: int = 10):
```

Comment:
place unit of measurement for time variables: is this ms or second or something else?

Reply:
Done